docs(mdx): Add clearer instructions on the storage location of mdx-components files. (#52187)
vercel/next.js: docs/02-app/01-building-your-application/06-configuring/05-mdx.mdx

npm install @next/mdx @mdx-js/loader @mdx-js/react @types/mdx
```

> Create `mdx-components.tsx` in the root of your application (the parent folder of `app` if without use src directory):

Reviewer: ```suggestion
Create `mdx-components.tsx` in the root of your application (the parent folder of `app/` or `src/`):
```
